src/server/systems/NetworkingServer.hpp

```diff
@@ -23,7 +23,7 @@
 #include "../../common/util/StateDef.hpp"
 
 class NetworkingServer {
-    const uint32_t CMD_DRIFT_THRESHOLD = 7;
+    const uint32_t CMD_DRIFT_THRESHOLD = 1000000;
```

Comment:
is this still used? seems like it should be deleted since we have `GAME_SETTINGS.INPUT_TICK_DELTA_THRESHOLD` now

Reply:
oops i thought i left it out